### ISSUE TITLE
A11y Master Bug

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/ioweb2016#110]
## General
- [ ] Offscreen, responsive side nav is focusable even when not displaying
- [ ] Unknown elements might be ripples? Not sure but they take over focus when you click on some links
## Masthead
- [x] No focus states for home button
- [ ] Navigation initially announces 3 items but there's actually 5
- [ ] No focus states for top level navigation
- [ ] No role for 'Sign in'
- [ ] 'Sign in' is not focusable
- [ ] Top level navigation announces 'unknown' in VO
- [ ] Hitting Enter on 'unknown' triggers ripple but page doesn't navigate
- [ ] Need to manage focus when selecting a menu item
## Home page
- [ ] Back to top button focusable even when its hidden
- [ ] Can focus outter and then inner back to top button. Inner button just announces 'button'
- [ ] Back to top button needs label
- [ ] Focus style on "Shoreline Amphitheatre Mountain View, CA" not visible against blue background
- [ ] Map link could maybe have better description
- [ ] Video button is not focusable
## Schedule
- [ ] Day 1 focuses first, only way to focus Agenda is shift+tab
- [ ] Enter on Day 1 unknown element doesn't work
- [ ] Clicking 'Browse Events' will need to manage focus
- [ ] Filters need aria-labelledby
- [ ] Should focus disappear after keyboard-pressing a filter button?
- [ ] Using a screenreader, after you navigate past Travel it stops on an invisible toolbar
- [ ] Event map overlay is a big can of worms :D
## I/O Extended
- [ ] Can't focus recap video button
